File: homework7/task2.js
const fs = require("fs");

// Read the list.json file
const rawData = fs.readFileSync("list.json", "utf-8");
const data = JSON.parse(rawData);

// Find all list objects and print their ID and name
console.log("Lists from ClickUp:");
data.lists.forEach((list) => {
  console.log(`ID: ${list.id} | Name: ${list.name}`);
});

File: homework9/task2.py
import json

# Read the lists.json file
with open("lists.json", "r") as file:
    data = json.load(file)

# Find all list objects and print their ID and name
print("Lists from ClickUp:")
for lst in data["lists"]:
    print(f"ID: {lst['id']} | Name: {lst['name']}")